README.md

```diff
@@ -718,16 +753,6 @@ Latest ✔ | Latest ✔ | 9+ Full ✔ | Latest ✔ | Latest ✔ |
 
 > v3.x removed ES3 / IE8, if you need to retain ES3 (IE8) compatibility you will need to remain on the v2.x versions of the SDK. Which is now maintained on the old [master branch](https://github.com/Microsoft/ApplicationInsights-JS/tree/master)
 
-### Build and Test this Project
-
-Note: With the recent update to the latest version of rush ```npm run build``` fails with exit code 1 on successful build, hence the addition of ```--silent``` to the ```npm run build``` command.
-
```

Comment:
this part redundant since we already includes the necessary information here (https://github.com/microsoft/ApplicationInsights-JS?tab=readme-ov-file#build-and-test-this-project)